### PR TITLE
Corrected Me Too link to Ask page

### PR DESCRIPTION
**Pull request by machavity**
The way it was done previously would add a $ into the URL

### CODE REVIEW

File: src/FlagTypes.ts [hunk 1]
```diff
@@ -110,7 +110,7 @@ export const flagCategories: FlagCategory[] = [
                 DisplayName: 'Me too',
                 ReportType: 'AnswerNotAnAnswer',
                 GetComment: () => 'Please don\'t add *"Me too"* as answers. It doesn\'t actually provide an answer to the question. ' +
-                    `If you have a different but related question, then [ask](//${window.location.hostname}$/questions/ask) it ` +
```
Comment (j-f1) [inline src/FlagTypes.ts]: You can just drop the second `$` here.

File: src/FlagTypes.ts [hunk 2]
```diff
@@ -110,7 +110,7 @@ export const flagCategories: FlagCategory[] = [
...
+                    `If you have a different but related question, then [ask](//stackoverflow.com/questions/ask) it ` +
```
Comment (j-f1) [inline src/FlagTypes.ts]: ...or change the URL to `/questions/ask`.

File: src/FlagTypes.ts (hunk 1 above)
Comment (machavity) [reply src/FlagTypes.ts]: None of the other links use that syntax, however. I was aiming for consistancy

**Comment by rjrudman** [comment]
Good catch, thanks!

**Pull request merged**